front/src/config/i18n/en.json

```diff
@@ -791,6 +791,18 @@
         "saveLabel": "Save configuration"
       }
     },
+    "oauth2": {
+      "buttonConnect": "Connect now",
+      "apiKeyLabel": "Then, enter your client_id and secret_id key here:",
+      "errorEmptyConfig": "Client Id and Secret Id must not be empty !",
+      "errorServiceId": "Service Id of current integration must not be found !",
+      "errorAuthorizationUri": "Error in authorization provider !",
```

Comment:
Not fan of this text, not clear to me.
For more, in english, no space before `!`.

Reply:
I change errorAuthorizationUri to be more explicit, normally this message never appear except if developper do an error in configuration file